### DISCUSSION TITLE
Install should use /usr/bin/stat  not stat

### DISCUSSION THREAD

**Discussion by DG12** [Homebrew/discussions#5376] [Getting started]
Custom version of stat may be inconsistent with standard version.
```
STAT_PRINTF=("stat" "-f")
STAT_PRINTF=("stat" "--printf")
```

### Output of `brew doctor`

```shell
stat: can't use format 'f' with 'f'
```

### Description of issue

Custom version of stat includes

/usr/bin/stat -t "%x %X " **-f**"%z %Sa %Sm %Sc %SB %N" $* | sed "s/\"//g ; s/$ccyymmdd/__-__-__/g" | column -t

**Comment by SMillerDev** [comment]
Install of what?
